SPECK/src/main/java/SPEck.java

```diff
@@ -227,19 +296,22 @@ protected static void writeDataset(String fileOut, Utils.Dataset dataset) throws
     /**
      * Compute the p-values of the FSP of a dataset merging the p-values computed by the different cores.
      * @param   T               the number of random datasets for the Monte Carlo estimate of p-values
-     * @param   parallelization the number of cores of the machine. It is the number of random datasets
+     * @param   numCores        the number of cores of the machine. It is the number of random datasets
      *                          generated at the same time in the Monte Carlo estimation. Each core of
-     *                          the machine computes the p-values on T/parallelization random datasets.
```

Comment:
Instead of "Each core of the ...", replace with:"It determines the size of the batches of the T datasets from which the partial sums for the p-values are computed, before being aggregated."